Libraries/acdriver/CMakeLists.txt

```diff
@@ -14,11 +14,17 @@ add_library(acdriver SHARED
             Sources/Result.cpp
             Sources/VersionAction.cpp
             Sources/CompileAction.cpp
+            Sources/CompileActionImageSet.cpp
+            Sources/CompileOutput.cpp
```

Comment:
Yeah, I'm not saying tests for the compile action are needed right now, that's pretty complex since it would need mock assets as an input. But I think the `CompileOutput` should get tests since it's relatively much simpler.